pkg/cloudprovider/external.go

```diff
@@ -33,6 +33,8 @@ func IsCloudProviderExternal(platformStatus *configv1.PlatformStatus, featureGat
 		return isExternalFeatureGateEnabled(featureGate)
 	case configv1.IBMCloudPlatformType:
 		return true, nil
+	case configv1.AlibabaCloudPlatformType:
+		return true, nil
```

Comment:
This might be tidier/easier for us to extend in the long term
```suggestion
	case configv1.IBMCloudPlatformType,
		configv1.AlibabaCloudPlatformType:
		return true, nil
```

Reply:
Fixed. Thanks for the suggestion.